cloudslang-tests/src/test/java/io/cloudslang/lang/systemtests/SimpleFlowTest.java

```diff
@@ -133,6 +138,21 @@ public void testFlowWithMissingNavigationFromOperationResult() throws Exception
     }
 
     @Test
+    public void testFlowWithRequiredInputUTF8() throws Exception {
+        URI resource = getClass().getResource("/yaml/flow_with_required_input.sl").toURI();
+        URI operations = getClass().getResource("/yaml/print.sl").toURI();
+        String inputValue = FileUtils.readFileToString(new File(getClass().getResource("/inputs/utf8_input.txt").getFile()),
+                StandardCharsets.UTF_8);
+        Map<String, Serializable> inputs = new HashMap<>();
+        inputs.put("input", inputValue);
+
+        SlangSource operationsSource = SlangSource.fromFile(operations);
+        Set<SlangSource> path = Sets.newHashSet(operationsSource);
+        CompilationArtifact compilationArtifact = slang.compile(SlangSource.fromFile(resource), path);
+        trigger(compilationArtifact, inputs, SYS_PROPS);
```

Comment:
you could propagate the value through Jython (use publish for task) and verify the flow output